book/en/src/by-example/tips/indirection.md

```diff
@@ -13,14 +13,14 @@ As this example of approach goes completely outside of RTIC resource model with
 
 Here's an example where `heapless::Pool` is used to "box" buffers of 128 bytes.
 
-``` rust,noplayground
-{{#include ../../../../../rtic/examples/pool.rs}}
+```rust,noplayground
+{{#include ../../../../../examples/lm3s6965/examples/pool.rs_old}}
```

Comment:
This looks like that chapter is outdated?

Reply:
@romancardenas Do you recall why `pool.rs` example got renamed to `pool.rs_old` as part of [PR815](https://github.com/rtic-rs/rtic/pull/815/files#diff-bfd7d183b2d78e4f0cb8d28f140120aa5428525ef9eba312cd6d89f2d83077e9)?

Reply:
No, my guess is that I did a refactor or something while moving the CI outside the rtic crate and I accidentally renamed the file.

Reply:
Then let's merge this, together with a renaming of the file. Skipping this last commit https://github.com/rtic-rs/rtic/pull/912/commits/d3692ca8b50d239d4cf8421a3ae73b828a49815b